lib/wings/active_fedora_converter.rb

```diff
@@ -120,54 +120,38 @@ def initialize(uri = RDF::Node.new, _parent = ActiveTriples::Resource.new)
       def convert_members(af_object)
         return unless resource.respond_to?(:member_ids) && resource.member_ids
         # TODO: It would be better to find a way to add the members without resuming all the member AF objects
-        ordered_members = []
-        resource.member_ids.each do |valkyrie_id|
-          ordered_members << ActiveFedora::Base.find(valkyrie_id.id)
-        end
-        af_object.ordered_members = ordered_members
+        af_object.ordered_members = resource.member_ids.each_with_object([]) { |valkyrie_id, arr| arr << ActiveFedora::Base.find(valkyrie_id.id) }
```

Comment:
```suggestion
        af_object.ordered_members = resource.member_ids.each_with_object([]) do |valkyrie_id, arr|
          arr << ActiveFedora::Base.find(valkyrie_id.id)
        end
```

or maybe just a `.map { ActiveFedora::Base.find(valkyrie_id.id) }`??

Reply:
rewrote to use .map